src/main/31-cloudfront.tf

```diff
@@ -72,21 +77,22 @@ module "fe_cdn" {
   }
   ordered_cache_behavior = [
     {
-      path_pattern             = "/eservices/*"
+      path_pattern             = "/eservices"
       target_origin_id         = "apigw"
       viewer_protocol_policy   = "redirect-to-https"
       cache_policy_id          = data.aws_cloudfront_cache_policy.caching_disabled.id
-      origin_request_policy_id = data.aws_cloudfront_origin_request_policy.all_viewer.id
+      origin_request_policy_id = data.aws_cloudfront_origin_request_policy.all_viewer_except_host_header.id
       allowed_methods          = ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"]
       compress                 = true
       use_forwarded_values     = false
+
     },
     {
-      path_pattern             = "/producers/*"
+      path_pattern             = "/producers"
```

Comment:
This will work only for the exact match `/producers`. 
Paths like `/producers/<producerId>` won't get routed towards the APIGW.

Reply:
same as above